packages/core/src/plugins/use-error-handler.ts

```diff
@@ -10,14 +10,16 @@ type ErrorHandlerCallback<ContextType> = {
 };
 
 const makeHandleResult =
-  <ContextType>(errorHandler: ErrorHandler) =>
+  <ContextType extends Record<any, any>>(errorHandler: ErrorHandler) =>
   ({ result, args }: ErrorHandlerCallback<ContextType>) => {
     if (result.errors?.length) {
       errorHandler(result.errors, args);
     }
   };
 
-export const useErrorHandler = <ContextType>(errorHandler: ErrorHandler): Plugin<ContextType> => {
+export const useErrorHandler = <ContextType extends Record<string, any>>(
```

Comment:
technically this is a breaking change 🤔

Reply:
but could also be considered as a bug fix, as the earlier type did not satisfy the usage constraint...